status-im/status-react
# feat: implement saved contact address list item component

fixes #17387  

### Summary

Implement Saved Contact Address List Item component

#### Screenshots

<img src="https://github.com/status-im/status-mobile/assets/18485527/193a56ff-2418-4090-85ca-4347ff6b95fd" width="360" /> <img src="https://github.com/status-im/status-mobile/assets/18485527/91e6c146-0dd9-4118-a826-e0f0d2b2952a" width="360" /> <img src="https://github.com/status-im/status-mobile/assets/18485527/6382c3fe-5fc3-4dcb-a4ac-e5501f8be507" width="360" /> <img src="https://github.com/status-im/status-mobile/assets/18485527/94248bc1-ad98-49da-95d3-dc4c4cd09886" width="360" />

#### Platforms

- Android
- iOS

### Steps to test

- Open Status
- Open Quo2 previews
- Select List Items > Saved Contact Address
- Verify correct design and behavior of the component

status: ready
